deployment/terraform/utils.go

```diff
@@ -114,3 +117,30 @@ func openBrowser(url string) (err error) {
 	}
 	return
 }
+
+func validateLicense(filename string) error {
```

Comment:
This sounds like a good candidate for a unit test?

Reply:
I believe most of the functionality is already covered by the tests in the server repository.
We could add a unit test but it would require including sample license files.

Reply:
Yeah, I think I would be fine with that. I mainly wanted to test the `!IsStarted` and `IsExpired` part, and verify that we are indeed returning errors for them. They are individually tested in the server repo, but this additional logic isn't tested.